src/home.js

```diff
@@ -61,7 +61,7 @@ var HomeDepartmentsAndPages = React.createClass({
   pages: ['Purcell-Cutts House', 'Provenance Research', 'Deaccessions', 'Conservation'],
 
   render() {
-    return <div className="landingPageBody">
+    return <main role="main" id="main" className="landingPageBody">
```

Comment:
Same concept as here: https://github.com/nlaffey/art/pull/2/files#r361749695

There's also likely other places on the site that could use this update